vendors/vendor/boards/board/aws_demos/application_code/main.c

```diff
@@ -156,8 +157,16 @@ int main( void )
                             tskIDLE_PRIORITY,
                             mainLOGGING_MESSAGE_QUEUE_LENGTH );
 
+    /* Initialize the AWS Libraries system.
+     * This initializes the CRYPTO random number state,
+     * so if your port is using FreeRTOS+TCP, it should
+     * be called before FreeRTOS_IPInit().  Any hardware entropy
+     * initialization must be done before calling SYSTEM_Init.*/
+    SYSTEM_Init();
```

Comment:
We're showing this call in main in our reference starter code, but some of our "real" ports do it in the RTOS daemon callback. Can we explain why customers should choose one or the other?